README.md

```diff
@@ -63,6 +63,12 @@ bin/cake migrations mark_migrated 20150417223600
 # Since Migrations 1.3.1, a new `all` special value for the version argumentwas added.
 # The following will mark all migrations found as migrated.
 bin/cake migrations mark_migrated all
+
+# The following option `--target` will try to mark the migrations from beginning to the given version (including it)
+bin/cake migrations mark_migrated 20150417223600 --target
+
+# You can use the option `--exclude` along with `--target` and it will try to mark the migrations from beginning until the given version (excluding it)
+bin/cake migrations mark_migrated 20150417223600 --target --exclude
```

Comment:
These options don't make sense in context of how Phinx works.

Should rather be:
`bin/cake migrations mark_migrated` instead of `bin/cake migrations mark_migrated all`
`bin/cake migrations mark_migrated --target=20150417223600 ` instead of `bin/cake migrations mark_migrated 20150417223600 --target`
`bin/cake migrations mark_migrated --target=20150417223600 --exclude ` instead of `bin/cake migrations mark_migrated 20150417223600 --target --exclude`

With Phinx, the target (when migrating) is always assumed to be the latest version unless a `--target` is specified